sql-statements/sql-statement-rollback.md

```diff
@@ -35,7 +35,7 @@ Empty set (0.01 sec)
 
 ## MySQL compatibility
 
-This statement is understood to be fully compatible with MySQL. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
+This statement is partly compatible with MySQL. Closing the connection or continuing to open a new transaction after the transaction is rolled back is not supported. Any compatibility differences should be [reported via an issue](https://github.com/pingcap/tidb/issues/new/choose) on GitHub.
```

Comment:
Can you please clarify this statement? I can't make sense out of it.

Reply:
I decided to remove this, and just say "TiDB does not support the syntax `ROLLBACK TO SAVEPOINT`. Any compatibility differences should be [reported via an issue](https://github.com/pingcap/tidb/issues/new/choose) on GitHub." I will update it in master in https://github.com/pingcap/docs/pull/3664